aws-ec2/README.md

```diff
@@ -72,6 +73,68 @@ For more information about the filters, see <a target="_blank" href="http://docs
 
 For more information about the metrics emitted by Amazon EC2, visit the service's homepage at <a target="_blank" href="https://aws.amazon.com/ec2/">https://aws.amazon.com/ec2/</a>.
 
+### RECOMMENDED Metrics
+
+The following are a subset of available metrics; these are recommended by Amazon for collection.
+
+
+_Standard EC2 Metrics_
+
+CPUUtilization
```

Comment:
Many of these metrics do not list a recommended stat. I asked @asuresh4 about this and this implies that all 5 stats are collected. My recommendation would be to explicitly state that for each metric so there is no confusion (as it's entirely possible somebody would not read a comment at the top of the list or a general comment on all metrics).

Reply:
Done.